README.md

```diff
@@ -48,16 +48,36 @@ for 'hello world':
 
     kubectl expose deployment appka --type=LoadBalancer --name=appka-service
 
+You can inspect the Akka Cluster membership status with the [Cluster HTTP Management](https://doc.akka.io/docs/akka-management/current/cluster-http-management.html).
+
+    curl http://127.0.0.1:8558/cluster/members/
+
 To check what you have done in Kubernetes so far, you can do:
 
     kubectl get deployments
     kubectl get pods
     kubectl get replicasets
     kubectl cluster-info dump
+    kubectl logs appka-79c98cf745-abcdee   # pod name
     
 To wipe everything clean and start over, do:
 
     kubectl delete namespaces appka-1
+    
+## Running in a real Kubernetes cluster
+
+#### Publish to a registry the cluster can access e.g. Dockerhub with the kubakka user
+
+The app image must be in a registry the cluster can see. The build.sbt uses DockerHub by default.
+Use `mvn -Ddocker.registry=$DOCKER_REPO_URL/$NAMESPACE` if your cluster can't access DockerHub.
+  
+To push an image to docker hub run:
+
+    mvn -am -pl bootstrap-demo-kubernetes-api package docker:push 
```

Comment:
I haven't tried this, but we can verify when we write the deployment guide. See https://developer.lightbend.com/guides/openshift-deployment/lagom/building-using-maven.html